diff --git a/libs/doomgeneric/doomgeneric/am_map.c b/libs/doomgeneric/doomgeneric/am_map.c
index 5d9f7e2..8d9f7e3 100644
--- a/libs/doomgeneric/doomgeneric/am_map.c
+++ b/libs/doomgeneric/doomgeneric/am_map.c
@@ -424,7 +424,7 @@
 void AM_initVariables(void)
 {
     int pnum;
-    static event_t st_notify = { ev_keyup, AM_MSGENTERED, 0, 0 };
+    static event_t st_notify = { ev_keyup, AM_MSGENTERED, 0, 0, 0 };
 
     automapactive = true;
     fb = I_VideoBuffer;
@@ -540,7 +540,7 @@
 //
 void AM_Stop (void)
 {
-    static event_t st_notify = { 0, ev_keyup, AM_MSGEXITED, 0 };
+    static event_t st_notify = { 0, ev_keyup, AM_MSGEXITED, 0, 0 };
 
     AM_unloadPics();
     automapactive = false;
@@ -1292,6 +1292,7 @@
 ( int	colors,
   int 	colorrange)
 {
+    (void) colorrange;
     int		i;
     mobj_t*	t;
 
